tests/unittests/decorators.py

```
@@ -20,6 +22,15 @@ def drop_datasets(func):
     before running a test.
     """
 
+    if iscoroutinefunction(func):
+
+        @wraps(func)
+        async def async_wrapper(*args, **kwargs):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `async_wrapper`

(ANN202)